src/components/Dialogs/ImageCropDialog/ImageCropDialog.tsx

```diff
@@ -83,15 +84,15 @@ const ImageCropDialogComponent: React.ForwardRefRenderFunction<
 
   const zoomControlNode = (
     <ZoomControl>
-      <Icon name="zoom-out" />
+      <StyledZoomButton icon="zoom-out" variant="tertiary" onClick={() => handleZoomChange(currentZoom - 0.1)} />
```

Comment:
These values may not work the best here. What works best will depend on the picture really, take a look:

https://user-images.githubusercontent.com/12646744/111460165-f2110300-871b-11eb-9775-de56734e6cf4.mov

The change should be more gradual here. Maybe you can use `zoomStep` for that?

Reply:
I wasn't aware of this, tested only on one image. Should be good now.